### PR TITLE
legacy mode merge

### PR DESCRIPTION
**Pull request by vetalcore**

### CODE REVIEW

**Review by mirceahasegan — APPROVED** [review]
lgtm

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**processReports**: ➖ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/7039/11121957484/index.html) for [be07d687](https://github.com/input-output-hk/lace/pull/1455/commits/be07d6874af796e4391725647b078c9f3a8a3772)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 0      | 0      | 0       | 0     | 0     | ➖     |

File: apps/browser-extension-wallet/package.json [hunk 1]
```diff
@@ -47,19 +47,22 @@
...
     "@lace/common": "0.1.0",
     "@lace/core": "0.1.0",
+    "@lace/nami": "^0.1.0",
     "@lace/staking": "0.1.0",
     "@lace/translation": "0.1.0",
     "@pdfme/generator": "^4.0.2",
     "@react-rxjs/core": "^0.9.8",
     "@react-rxjs/utils": "^0.9.5",
     "@shiroyasha9/axios-fetch-adapter": "^1.0.3",
+    "@xsy/nami-migration-tool": "file:./xsy-nami-migration-tool-0.0.39.tgz",
```
Comment (pczeglik-iohk) [inline apps/browser-extension-wallet/package.json:65]: is this reference ok?

**Review by pczeglik-iohk — APPROVED** [review]

File: apps/browser-extension-wallet/package.json (hunk 1 above)
Comment (vetalcore) [reply apps/browser-extension-wallet/package.json:65]: would be handled as part of lw-11435

Comment (pczeglik-iohk) [reply apps/browser-extension-wallet/package.json:65]: ok

**Pull request merged**